Release/include/cpprest/http_client.h

```diff
@@ -347,6 +348,25 @@ class http_client_config
     {
         return m_ssl_context_callback;
     }
+
+    /// <summary>
+    /// Gets the TLS server name indication (SNI) property.
+    /// </summary>
+    /// <returns>True if TLS server name indication is enabled, false otherwise.</returns>
+    bool tlsext_host_name() const
+    {
+        return m_tlsext_host_name;
+    }
+
+    /// <summary>
+    /// Sets the TLS server name indication (SNI) property.
+    /// </summary>
+    /// <param name="tlsext_host_name">False to disable the TLS (ClientHello) extension for server name indication, true otherwise.</param>
+    /// <remarks>Note: This setting is required in most virtual hosting scenarios.</remarks>
+    void set_tlsext_host_name(bool tlsext_host_name)
```

Comment:
same as above, this is not setting the host name but enabling the set_tlsext_host_name feature.